index.html

```diff
@@ -4,9 +4,49 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+    <link rel="stylesheet" href="./src/styles.css">
+    <link rel="stylesheet" href="./src/home/index.css">
+    <title>Portland People Quest</title>
 </head>
 <body>
-    
+    <header>
+        <h1>Portland People Quest</h1>
+    </header>
+    <main>
+        <h2>Are you ready to start your Portland adventure?</h2>
+        <form id="portlander-sign-up">
+            <p>
+                <label for="name">Enter your name:</label>
+                <input id="name" name="name" required>
+            </p>
+            <fieldset>
+                <legend>Choose your adventurer!</legend>
+                <p id="avatar">
+                    <label>
+                        <input type="radio" name="portlander" required value="student">
+                        Student
+                        <img src="./assets/student.svg" alt="">
```

Comment:
Don't forget to put something descriptive in your alt attribute. No need to say 'image of..' because the screen reader will say that it's an image.